ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -94,7 +94,10 @@ if [[ -n "${TEST_SKIPS}" ]]; then
     TEST_ARGS="${TEST_ARGS:-} --file /tmp/tests"
 fi
 
-openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" ${TEST_ARGS:-} \
+# using set -x so we will see the full openshift-tests command in the log. disable right after.
+set -x
+openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" "${TEST_ARGS:-}" \
```

Comment:
We can have multiple arguments in `TEST_ARGS`.  We can build it as a Bash array if we need to keep ShellCheck happy, or just leave master's unquoted expansion alone.

Reply:
my bad. the online shellcheck tool I was using flagged this so I blindly just quoted it. our validation doesn't seem to
care though so no tricks needed. just force pushed this PR to remove quotes from `TEST_ARGS`